vrecord

```diff
@@ -1286,6 +1351,8 @@ if [[ "${QCTOOLSXML_CHOICE}" != "No" ]] ; then
     if [[ "${BRNG_OUTLIERS}" -gt "${BRNG_OUTLIER_THRSHLD}" ]] ; then
         cowsay "$(_report -w "WARNING: Your video file contains ${BRNG_OUTLIERS} frames with pixels out of broadcast range.")"
     fi
+    _report -d "Vrecord is generating graphs from the QCTools data. One moment please."
+    _qcgraphimage "${QCXML}"
```

Comment:
I might move this above the `else` at 1342 so that the function is run only on successful confirmation of `${QCXML}`